packages/integration-karma/test/component/LightningElement.disconnectedCallback/index.spec.js

```diff
@@ -157,3 +159,94 @@ it('should associate the component stack when the invocation throws', () => {
     expect(error.message).toBe('throw in disconnected');
     expect(error.wcStack).toBe('<x-disconnected-callback-throw>');
 });
+
+describe('disconnectedCallback for components with a custom render()', () => {
+    it('should invoke disconnectedCallback for children when parent is removed', () => {
```

Comment:
is this test related to the issue? seems like a regular test.

Reply:
Yes, this is a regular test for a component with a custom render function. Added it since there was no test for this scenario.